KryvaAnastasiia/HW7/task3.py

```diff
@@ -0,0 +1,11 @@
+word = input("Enter any word :")
+
+def number_of_characters(word):
```

Comment:
look at more optimized version of the code
def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result